ports/libsamplerate/portfile.cmake

```
@@ -1,24 +1,27 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO libsndfile/libsamplerate
-    REF f6730d03c3e7660bb6ecad8816f1b09c5825142a # v0.1.9
-    SHA512 6a349c9144d024212fc78dc0d9e39bdc1a43abaf590fcfbf84396af22834545962d5ef10176b48b21fcae2ce62d12277b682059383805d059f4dd2b9f6708478
+    REF 0.2.0
+    SHA512 6a12506ad391a557fcc354c7bb5767ae71349da26bf31753b806a36449cfc879ef97e4d25adab5722a9540ec1539203c6155b4581e42bdb8b6708dca1f089d11
     HEAD_REF master
-    PATCHES Use-the-lrintf-intrinsic.patch
 )
 
-file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH}/src)
-file(COPY ${SOURCE_PATH}/Win32/config.h DESTINATION ${SOURCE_PATH}/src)
-
 vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}/src
+    SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
+    OPTIONS
+        -DBUILD_TESTING=OFF
+        -DLIBSAMPLERATE_EXAMPLES=OFF
 )
 
 vcpkg_install_cmake()
 
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/SampleRate TARGET_PATH share/SampleRate)
+vcpkg_fixup_pkgconfig(SYSTEM_LIBRARIES m)
```

```suggestion
vcpkg_fixup_pkgconfig()
```
This argument has invalided now. So please remove it.